### PR TITLE
[#203]-added button group

### PR DESCRIPTION
**Pull request by tanujary**
WIP

### CODE REVIEW

File: src/components/ux-button-group/ux-button-group.feature [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-button-group test suite
+
+  Scenario: Loading ux-button-group
+    Given I have loaded component "ux-button-group" with use case "UseCase"
```
Comment (pvshum) [inline src/components/ux-button-group/ux-button-group.feature]: @tanujary Please provide some tests.

**Comment by tanujary** [comment]
Can someone look into my pull request? :)

File: src/components/ux-button-group/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+> ux-button-group Documentation
```
Comment (nathf) [inline src/components/ux-button-group/README.md:1]: Need to write some documentation

File: src/components/ux-button-group/ux-button-group.feature [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Feature: ux-button-group test suite
+@WIP
```
Comment (nathf) [inline src/components/ux-button-group/ux-button-group.feature]: Remove this WIP tag

File: src/components/ux-button-group/ux-button-group.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<ul id="{{guid}}" class="button-group {{class}}">
+	{{#if isDataModel}}
+		{{#each items}}
+			<ux-button-groupitem datamodel="{{{.}}}" />
+		{{/each}}
+	{{else}}
+		{{>content}}
```
Comment (nathf) [inline src/components/ux-button-group/ux-button-group.hbs:7]: We should `{{yield}}` in favor of `{{>content}}`

File: src/components/ux-button-group/ux-button-group.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Ractive.extend({
+	template: Ractive.defaults.templates['ux-button-group'],
+
+	isolated: true,
+	
+	computed: {
+		guid: function () {
+			return this._guid;
+		}
```
Comment (nathf) [inline src/components/ux-button-group/ux-button-group.js:9]: Whats this function for?

File: src/components/ux-button-groupitem/ux-button-groupitem.hbs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<li>
+	{{#if isDataModel}}
+		<ux-button href="{{.href}}">{{.text}}</ux-button>
+	{{else}}
+		{{>content}}
```
Comment (nathf) [inline src/components/ux-button-groupitem/ux-button-groupitem.hbs]: Use `{{yield}}`

File: src/components/ux-button-group-item/ux-button-group-item.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+Ractive.extend({
+	template: Ractive.defaults.templates['ux-button-group-item'],
+	isolate: true
```
Comment (eneo5541) [inline src/components/ux-button-group-item/ux-button-group-item.js:3]: Change 'isolate' to 'isolated' for consistency.

**Comment by eneo5541** [comment]
Is it really necessary to define a new ux-button-group-item that is essentially a wrapper for the ux-button component?

**Pull request closed without merging**